AirCasting/SessionViews/ABMeasurementsView.swift

```diff
@@ -110,7 +117,7 @@ struct _ABMeasurementsView: View {
                 streamsToShow.count != 1 ? Spacer() : nil
                 ForEach(streamsToShow.filter({ !$0.gotDeleted }), id : \.self) { stream in
                     SingleMeasurementView(stream: stream,
-                                          threshold: nil,
+                                          threshold: .init(value: nil),
```

Comment:
Here you don't need to define the value as nil. The reason is the init so:

```
 init(value: SensorThreshold? = nil) {
         self.value = value
     }
```
You have a default value that is the same as the one you are assigning here.